src/server/apsystem.ts

```diff
@@ -432,20 +624,41 @@ export default class ActivityPubSystem {
     }
 
     if (activity.actor !== undefined && typeof activity.actor === 'string') {
-      const interactedActorMention = await this.actorToMention(activity.actor, fromActor)
+      const interactedActorMention = await this.actorToMention(
+        activity.actor,
+        fromActor
+      )
       await actorStore.interacted.add([interactedActorMention])
     }
   }
 
-  async storeObject (fromActor: string, object: APObject, attributedTo: string = ''): Promise<void> {
-    if ((attributedTo.length !== 0) && (object.attributedTo !== attributedTo)) {
+  async storeObject (
+    fromActor: string,
+    object: APObject,
+    attributedTo: string = ''
+  ): Promise<void> {
+    if (attributedTo.length !== 0 && object.attributedTo !== attributedTo) {
       // TODO Shuld this be a different error? Should we just skip?
-      throw createError(419, `Unexpected author for object in activity. Expected ${attributedTo}, got ${object.attributedTo as string}. In object at ${object.id as string}`)
+      throw createError(
+        419,
+        `Unexpected author for object in activity. Expected ${attributedTo}, got ${object.attributedTo as string}. In object at ${object.id as string}`
+      )
     }
     const actorStore = this.store.forActor(fromActor)
     await actorStore.inboxObjects.add(object)
   }
 
+  async backfillOutbox (fromActor: string, toActor: string): Promise<void> {
+    const actorURL = await this.mentionToActor(fromActor)
+    const actor = await this.getActor(actorURL, fromActor)
+
+    const { outbox, id } = actor
+
+    for await (const activity of this.iterateCollection(fromActor, outbox)) {
```

Comment:
skip unreacheable activities?

Reply:
What is the advantage in skipping instead of stopping? I imagine any unreachable activities signify an error in the outbox. THo I suppose they might also be deleted/undone activities?

Reply:
Yeah ok, the outbox will be filled with Create/Update/Delete activities at least for the current behaviour of the jekyll-activity-pub plugin